java/serve/src/main/java/io/ray/serve/config/AutoscalingConfig.java

```diff
@@ -0,0 +1,101 @@
+package io.ray.serve.config;
+
+import java.io.Serializable;
+
+public class AutoscalingConfig implements Serializable {
+
+  private static final long serialVersionUID = 9135422781025005216L;
```

Comment:
Can we add a comment explaining where this number comes from?

Reply:
It is a hash value generated by IDE. Both default value 1L and hash value are OK here. We find the serialVersionUID in Java ray-api and ray-runtime is hash. So we unified the serialVersionUID  style in Java ray-serve.

Reply:
@shrekris-anyscale  It's a unique number, used on the deserializing end to make sure the classes is compatible.

 See https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/io/Serializable.html document for details: 
The serialization runtime associates with each serializable class a version number, called a serialVersionUID, which is used during deserialization to verify that the sender and receiver of a serialized object have loaded classes for that object that are compatible with respect to serialization. If the receiver has loaded a class for the object that has a different serialVersionUID than that of the corresponding sender's class, then deserialization will result in an [InvalidClassException](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/io/InvalidClassException.html). A serializable class can declare its own serialVersionUID explicitly by declaring a field named "serialVersionUID" that must be static, final, and of type long: